### DISCUSSION TITLE
淘宝小程序安卓真机上不显示文本

### DISCUSSION THREAD

**Discussion by SongBana** [galacean/discussion#10] [Mini Program（小程序）] [answered]
用createapp脚手架创建了一个淘宝小程序的项目，使用到了TextRender，模拟器上没问题，但是在安卓真机上显示不出来
demo地址：https://github.com/SongBana/toasis
淘宝IDE：https://developer.taobao.com/?spm=a219a.15212435.0.0.61a3669aW0JW9L

**Comment by gz65555** [comment]
@eyworldwide 有认识淘宝小程序的人咨询一下？真机和模拟器不一致的问题

**Comment by singlecoder** [comment]
已经定位问题，并在 galacean 修复

**Reply by eyworldwide** [reply]
PR 贴一下，版本号发布后也说一下。

**Comment by singlecoder** [comment, marked as answer]
修复 PR：https://github.com/galacean/engine/pull/1499
可在 0.8 最新版本中验证
